includes/admin/views/setup-wizard/step-coupon.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Setup Wizard step: Coupon
+ *
+ * @since [version]
+ * @version [version]
+ *
+ * @property LLMS_Admin_Setup_Wizard $this Setup wizard class instance.
+ */
+
+defined( 'ABSPATH' ) || exit;
+?>
+
+<h1><?php _e( 'Help Improve LifterLMS & Get a Coupon', 'lifterlms' ); ?></h1>
+<p><?php _e( 'By allowing us to collect non-sensitive usage information and diagnostic data, you\'ll be providing us with information we can use to make the future of LifterLMS stronger and more powerful with every update!', 'lifterlms' ); ?></p>
+<p><?php _e( 'Click "Allow" to and we\'ll send you a coupon immediately.', 'lifterlms' ); ?></p>
```

Comment:
I know this is just a c&p but since we're refactoring...
I think we somewhat talked about avoiding the use of the escaped quotes in favor of double quotes, probably to avoid translation issues?! Is it a fake memory?
I'm not going to pollute this review with this same comment on other occurrences in this PR, unless you want me doing it :D

Reply:
You're correct, that was not a fake memory, we did talk about that. Let's not worry about this here though. We can tackle that at some other time.